src/main.js

```diff
@@ -140,22 +140,24 @@ function updateStar(event) {
   return ideaId;
 };
 
-function viewStarredIdea() {
+function storeStar(event) {
+  var starID = starImage(event);
+  for (var i = 0; i < loggedIdea.length; i++) {
+    if (parseInt(starID) === loggedIdea[i].id) {
+      loggedIdea[i].isStarred = true;
+    }
+  }
+};
 
+function viewStarredIdea() {
+// conditional for loggedIdea[i].isStarred === true 
```

Comment:
Thanks for leaving these comments in here! Appreciate the start on the pseudocode!